donkeycar/templates/donkey2futucar.py

```diff
@@ -132,6 +133,10 @@ def drive_mode(mode,
     tub = TubWriter(path=cfg.TUB_PATH, inputs=inputs, types=types)
     V.add(tub, inputs=inputs, run_condition='recording')
 
+    # hall sensor rpm
+    rpm_sensor = HallSensorRpm(sensor = 17, wheel_radius = 33)
```

Comment:
Maybe not add directly to the vehicle-object since it's untested? rather read from config if it's enabled or not, and default should probably be false until we've tested it.